src/components/tooltip/tooltip.utils.js

```diff
@@ -0,0 +1,35 @@
+const oppositeDirections = {
+  left: 'right',
+  right: 'left',
+  top: 'bottom',
+  bottom: 'top'
+};
+
+function hasTheSameLine(tooltipPosition, pointerAlign) {
```

Comment:
`direction` will suggest that they have the same direction, but we will have to check if they reside on the same line, e.g.: top/bottom or left/right

Reply:
Ah OK. So maybe `haveTheSameOrientation()` would be a better name? Since the function will return `true` for all these pairs of inputs:
```
position: top,    align: top

position: top,    align: bottom

position: bottom, align: top

position: bottom, align: bottom

position: left,   align: left

position: left,   align: right

position: right,  align: left

position: right,  align: right
```
For example, with `position: top, align: bottom`, they both have the same orientation (vertical).

Reply:
I was looking for a proper word, looks like orientation will be fine :)

Reply:
Many thanks for renaming it :slightly_smiling_face: